### DISCUSSION TITLE
Source code of example

### DISCUSSION THREAD

**Discussion by rodolphoasb** [clauderic/dnd-kit#781] [Q&A] [answered]
Hey, @clauderic! How are you doing?

Would you be able to share the source code of this example https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-vertical--drag-handle?

I cannot find it inside the stories...

**Comment by clauderic** [comment, marked as answer]
The stories are optimized for re-usability and maintainability. Ideally there would be isolated code examples for every single story, but this isn't something I have bandwidth to work on or maintain at the moment.

I created a starter template for @dnd-kit sortable on CodeSandbox which you can refer to: https://codesandbox.io/s/22x1ix
